TESTING.md

```diff
@@ -14,33 +14,81 @@ Note that because the logs are from GNOME Shell, there will be messages from all
 
 ## Checklists
 
-Tasks for a tester to verify when approving a patch
+Tasks for a tester to verify when approving a patch. Use complex window layouts and at least two displays. Turn on active hint during testing.
+
+## With tiling enabled
+
+### Tiling
+
+- [ ] Super direction keys changes focus in the correct direction
+- [ ] Windows moved with the keyboard tile into place
+- [ ] Windows moved with the mouse tile into place
+- [ ] Windows swap with the keyboard (test with different size windows)
+- [ ] Windows can be resized with the keyboard (Test resizing four windows above, below, right, and left to ensure shortcut consistency)
```

Comment:
We may want to add something about trying to resize windows that are in different positions. I assume a grid of 4 and trying each quadrant would be sufficient.

Reply:
That's already there
- [ ] Windows can be resized with the keyboard (Test resizing four windows above, below, right, and left to ensure shortcut consistency)

Reply:
Ah, misinterpreted the directions as "resize the windows in these directions". Sounds good then.